src/app/action-items/action-items.component.ts

```diff
@@ -14,38 +16,41 @@ export interface ActionItemMapped extends ActionItem {
   templateUrl: './action-items.component.html',
   styleUrls: ['./action-items.component.css']
 })
-export class ActionItemsComponent implements OnInit {
+export class ActionItemsComponent implements OnInit, OnDestroy {
   dataSource: ActionItemMapped[];
-  isloadingActionItems = false;
-  private currentDate: Date = new Date();
+  isLoadingActionItems = false;
+  isLoadingActionItem = false;
+  actionItemId: string | undefined;
+  actionItem: ActionItem;
+  errorMessage: string | undefined;
+  justAddedItemId: string;
+  private querySubscription: Subscription;
+  private itemRequest: Subscription;
+  public ngxScrollToDestination: string;
```

Comment:
Unused property. Also in TS all properties are public by default so there is no need to add it explicit.